lib/ansible/modules/packaging/language/pip.py

```diff
@@ -5,6 +5,21 @@
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import absolute_import, division, print_function
+
+import operator
+import os
+import re
+import shlex
+import sys
+import tempfile
+import traceback
+from distutils.version import LooseVersion
+
+from ansible.module_utils._text import to_native
+from ansible.module_utils.basic import (AnsibleModule, is_executable,
+                                        missing_required_lib)
+from ansible.module_utils.six import PY3
```

Comment:
These imports need to be moved back to where they were.  As you can see moving it has caused CI to fail.